sleep-for-days.md

```diff
@@ -0,0 +1,81 @@
+### Go
+---
+```
+// Send an email every 30 days
+func SleepForDaysWorkflow(ctx workflow.Context) (string, error) {
+  for {
+    workflow.ExecuteActivity(ctx, SendEmail) // Activities have timeouts, and will be retried by default!
```

Comment:
Technically this will fail as written because of a lack of activity options, but probably don't need to change anything

Reply:
Yea I omitted this as it was omitted in the "old" samples as well. Was wondering if it would be useful to add a:
```
// configure an activity...
```
but decided against it. WDYT?

Reply:
See other comment on adding the `// ...` instead

Reply:
Yeah, I like that, moving `// ...` to before the loop